src/wallet/wallet.cpp

```diff
@@ -3374,7 +3374,7 @@ bool CWallet::InitLoadWallet()
                 block = block->pprev;
 
             if (pindexRescan != block)
-                return InitError(_("Prune: last wallet synchronisation goes beyond pruned data. You need to -reindex (download the whole blockchain again in case of pruned node)"));
+                return InitError(_("Prune: last wallet synchronization goes beyond pruned data. You need to -reindex (download the whole blockchain again in case of pruned node)"));
```

Comment:
My goal was to have a standard spelling internally, for things that would never be shown to the user (such as comments as well as localization strings)